Documentation/GettingStarted.md

```diff
@@ -23,16 +27,77 @@ import AEPSignal
+
+  // Enable debug logging
+  MobileCore.setLogLevel(level: .trace)
+
+  MobileCore.registerExtensions([Lifecycle.self, Identity.self, Signal.self], {
+    // Use the App id assigned to this application via Adobe Launch
+    MobileCore.configureWith(appId: "yourAppId")
+    if application.applicationState != .background {
+      // Only start lifecycle if the application is not in the background
+      MobileCore.lifecycleStart(additionalContextData: ["contextDataKey": "contextDataVal"])
+    }
+     
+  })  
+  return true
+}
 ```
-        MobileCore.registerExtensions([Lifecycle.self, Identity.self, Signal.self], {
-
-            // Use the App id assigned to this application via Adobe Launch
-            MobileCore.configureWith(appId: "yourAppID")
-            if application.applicationState != .background {
-                // only start lifecycle if the application is not in the background
-                MobileCore.lifecycleStart(additionalContextData: ["contextDataKey": "contextDataVal"])
-            }
-        })
+
+## Implement Lifecycle Metrics
+
+Lifecycle metrics is an optional, yet valuable feature provided by the Adobe Experience Platform SDK. It provides out-of-the-box, application lifecycle information about your app user. These metrics contain information on the app user's engagement lifecycle such as device information, install or upgrade information, session start and pause times, etc. You can also set additional lifecycle metrics.
+
+
+
+Start Lifecycle data collection by calling `lifecycleStart:` from within the callback of `AEPCore.registerExtensions:`.
+
+```swift
+func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+
+  // Enable debug logging
+  MobileCore.setLogLevel(level: .trace)
+
+  MobileCore.registerExtensions([Lifecycle.self, Identity.self, Signal.self], {
+    // Use the App id assigned to this application via Adobe Launch
+    MobileCore.configureWith(appId: "yourAppId")
+    if application.applicationState != .background {
+      // Only start lifecycle if the application is not in the background
+      MobileCore.lifecycleStart(additionalContextData: ["contextDataKey": "contextDataVal"])
+    }
+     
+  })  
+  return true
+}
+```
+
+When launched, if your app is resuming from a backgrounded state, iOS might call your `applicationWillEnterForeground:` delegate method. You also need to call `lifecycleStart:`, but this time you do not need all of the supporting code that you used in `application:didFinishLaunchingWithOptions:`:
+
+```swift
+func applicationWillEnterForeground(_ application: UIApplication) {   
+  AEPCore.lifecycleStart(nil)
+}
 ```
 
+When the app enters the background, pause Lifecycle data collection from your app's `applicationDidEnterBackground:` delegate method:
+
+```swift
+func applicationDidEnterBackground(_ application: UIApplication) {   
+  AEPCore.lifecyclePause()
+}
+```
```

Comment:
should also add the sample code if the app is using App Scene. Don't think we need to do it now, but later we will also need sample code for the new SwiftUI lifecycle.